src/main/kotlin/inc/evil/d17map/exceptions/exceptions.kt

```diff
@@ -28,6 +28,8 @@ class UserNotFoundException : RuntimeException {
 class InvalidClassroomDataException
     : RuntimeException("Invalid classroom data: name cannot be blank and capacity must be greater than 0.")
 
+class InvalidExtraRoomDataException
```

Comment:
Jeśli używane tylko w ifie w kontrolerze to do wyrzucenia i tak samo od linijki 46